dev/conductor/core/test/packages_autoroller_test.dart

```diff
@@ -293,10 +293,6 @@ void main() {
         '-b',
         'packages-autoroller-branch-1',
       ]),
-      const FakeCommand(command: <String>[
```

Comment:
We only called this to ensure the tool was built, but the next command was calling the bash script that would ensure its built anyway, so this was redundant.